File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-01/plagiarized/L6/04/WelcomeToJava.kt
package IR

/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
object WelcomeToJava {
    @JvmStatic
    fun test(i: Int): String {
        return if (i > 0) {
            println("Welcome To Java")
            test(i - 1)
        } else {
            ""
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val i = 5
        test(i)
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-01/plagiarized/L5/03/WelcomeToJava.ts
import { Console } from 'console';

function test(i: number): void {
  while (i > 0) {
    Console.log('Welcome To Java');
    i--;
  }
}

function main(args: string[]): void {
  test(5);
}

main(process.argv.slice(2));

